docs/tutorial/pre-requisites-of-running-on-chain-spam-protected-chat2.md

```diff
@@ -16,96 +19,89 @@ If you already have an Ethereum account with sufficient ethers on the Sepolia te
     
     ![](https://i.imgur.com/zFduIV8.jpg)
 
-4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+4 The automatically created account points to the Ethereum Mainnet by default (dropdown menu in the top right corner).
     
     ![](https://i.imgur.com/gk3TWUd.jpg)
 
-5. You can use the same account for different networks. For Waku we need to connect to the Sepolia test network.
-6. You can switch to a test network by clicking on the drop down menu. Select Sepolia test network for Waku:
+   You can use the same account for different networks. To publish to the Waku Network, we need to connect to the Sepolia test network.
+5. Switch to the Sepolia test network by selecting it from the dropdown menu. Ensure "Show test networks" is enabled.
 
-    ![](https://i.imgur.com/kbuup6y.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/670778eb-8bf0-42a6-8dd7-1dedfabeeb37)
 
-7. Click on Show/hide test networks.
-8. Enable “Show Test Networks".
+   The same account can be used with different networks. Note that the ETH balance is different for each network (each uses a different token).
     
-    ![](https://i.imgur.com/02x7iSX.jpg)
-    
-    ![](https://i.imgur.com/UFsoRHR.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/0a5aa3a7-359c-4f4b-bd12-bad7c4844b34)
 
-9. Close the settings and now you should see the test networks in the drop down on the top right.
+6. To export your private key for this account, click on the three dots next to the account name and select "Account Details".
     
-    ![](https://i.imgur.com/leSYt4z.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/83fffa23-4a3b-46f9-a492-9748bfd47cff)
 
-10. Switch to Sepolia Test Network.
-11. You can see that the same account can be used with different networks. But note that the ETH balance on different networks are different and are different tokens.
+   Select "Show Private Key".
     
-    ![](https://i.imgur.com/Y7YkuEe.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/3a513389-2df1-4e32-86da-a1794126cdac)
 
-12. Now to export your private key for the account, if needed, please click on the three dots beside the account.
+   Enter your Metamask password and click "Confirm"
     
-    ![](https://i.imgur.com/9BlBmhT.jpg)
+    ![image](https://github.com/waku-org/nwaku/assets/68783915/ffbac631-b933-4292-a2c6-dc445bff153c)
 
-13. Click on Account Details.
-14. Click on Export Private Key.
-    
-    ![](https://i.imgur.com/7g6SyuX.jpg)
+   You will be shown the private key. Copy it as needed.
```

Comment:
What does "as needed" mean here? I'd add a note on a) where this key will be pasted and why; b) why this is not a security concern (I hope it's not).

Reply:
Very good point. This is within the context of a convenience tool for RLN registration that uses this key only for a registration on the testnet. Preferably the context in which this guide is linked should explain the context and dangers. However, I've added some context and warning in https://github.com/waku-org/nwaku/pull/2206/commits/4d921d9426bd22664ee3cf19ea355fb1219fe890.